src/pages/saksbehandling/sakintro/AvslutteBehandling.tsx

```diff
@@ -0,0 +1,103 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { useFormik } from 'formik';
+import { AlertStripeSuksess } from 'nav-frontend-alertstriper';
+import { Fareknapp } from 'nav-frontend-knapper';
+import { Select } from 'nav-frontend-skjema';
+import React, { useState } from 'react';
+
+import { slettBehandlingForSak } from '~features/saksoversikt/sak.slice';
+import * as Routes from '~lib/routes';
+import yup from '~lib/validering';
+import { useAppDispatch, useAppSelector } from '~redux/Store';
+
+export enum AvsluttetBegrunnelse {
+    VelgBegrunnelse = 'Velg begrunnelse',
```

Comment:
Är det ikke rart att denne ligger under `AvsluttetBegrunnelse`?

Reply:
ja, jeg tenkte litt på det, og ville ikke helt ha 2 typer, 1 som går på formen, og 1 for innsending. tenkte at validering av frontend og backend på det var kanskje litt mindre jobb.

Reply:
Hmm, ja sant. Men kanske man kunde representert 'Velg begrunnelse' som `null`? Alltså att man har `AvsluttetBegrunnelse | null`. Då tänker jag att definitionen av `AvsluttetBegrunnelse` blir mer riktig 😊 .